src/aiai_eval/cli.py

```diff
@@ -76,6 +76,20 @@
     show_default=True,
     help="Whether extra input should be outputted during benchmarking",
 )
+@click.option(
+    "--track-carbon-emissions",
+    "-tce",
+    is_flag=True,
+    show_default=True,
+    help="Whether to track carbon usage.",
```

Comment:
Maybe add here that `country-iso-code` should be specified?

Reply:
Seems reasonable. Will do.